fastapi_app/api/settings.py

```diff
@@ -0,0 +1,39 @@
+from datetime import timedelta
+
+from pydantic import (
+    BaseSettings
+)
+
+# Database settings
+DATABASE_URL = "postgresql+asyncpg://postgres:password@localhost:5432/postgres"
```

Comment:
Better to extract it into a .env file somehow, to set it up with docker-compose later

Reply:
Will be done in future